pkg/generate/app/cmd/newapp.go

```diff
@@ -117,6 +117,8 @@ type AppConfig struct {
 
 	OSClient        client.Interface
 	OriginNamespace string
+
+	ArgumentClassificationErrors map[string]error
```

Comment:
hmm ... yeah, slightly more cumbersome to use, but the implementation does prescribe a precedence.

I'll prototype and see if it looks palatable.